### ISSUE TITLE
Open-sourciness Score ("Forkability")

### ISSUE THREAD

**Issue by basicallydan** [opensauceit/ideas#8]
Inspired by #7 but more far-reaching! Want to encourage people to contribute to your project? Type in the GitHub URL and this tool will give you a score for how forkable your project is. Possible things to use for a measure:
- Presence of README
- README readability
- Presence of examples in README
- Presence of CONTRIBUTING
- Topics covered in CONTRIBUTING
- Average lifespan of an issue
- Pull request behaviour (lifespan/proportion pulled/closed/rejected/commented on) - this is a tricky one. Closed pull requests does not indicate unforkable.

Using these metrics we can suggest to the user how they can improve the forkability, with **examples** and best practices. These last two are vital. My hunch is that lack of examples is the biggest thing stopping people from doing things right. It certainly slows me down a lot of the time.

**TL;DR**: Google Webmaster Tools for GitHub Open Source projects.

**Comment by timruffles** [comment]
great idea

**Comment by petems** [comment]
Dont forget if there's a LICENSE.md file! :+1:

**Comment by ifraixedes** [comment]
I liked it and as a superset of my idea; I'm not sure if it should be in the same project, so creating different ones give the chance to the people to use or fork one of them to fit their needs.

Should I close my issue in favour of it?

**Comment by timruffles** [comment]
sounds like this is the frontend to your project linter

**Comment by ifraixedes** [comment]
Although it specifies the things to assess

**Comment by basicallydan** [comment]
Thanks! I can't make this month's Meetup, I'm on holiday. So if you make
any progress or chat about my idea pleaaaaase post it here :)

**Comment by ifraixedes** [comment]
Unfortunately, tomorrow I cannot make either.
I'm looking forward to being able to attend next time.

Enjoy

**Comment by basicallydan** [comment]
In case any of you lot are interested I decided to try this out: https://basicallydan.github.io/forkability

It's fairly simple at the mo in terms of what it checks, but I'd love some input :)
